README.md

```diff
@@ -1,14 +1,12 @@
-# natchez-http4s-otel - Opinionated Natchez Http4s Middlewares [![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.chrisdavenport/natchez-http4s-otel_2.12/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.chrisdavenport/natchez-http4s-otel_2.12) ![Code of Conduct](https://img.shields.io/badge/Code%20of%20Conduct-Scala-blue.svg)
-
-## [Head on over to the microsite](https://ChristopherDavenport.github.io/natchez-http4s-otel)
+# http4s-otel4s
 
 ## Quick Start
 
-To use natchez-http4s-otel in an existing SBT project with Scala 2.12 or a later version, add the following dependencies to your
+To use http4s-otel4s in an existing SBT project with Scala 2.12 or a later version, add the following dependencies to your
 `build.sbt` depending on your needs:
 
 ```scala
 libraryDependencies ++= Seq(
-  "io.chrisdavenport" %% "natchez-http4s-otel" % "<version>"
+  "org.http4s" %% "otel4s-middleware" % "<version>"
```

Comment:
To match [this line](https://github.com/http4s/http4s-otel4s/pull/23/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R47)?  (And consistent with other http4s modules...)

```suggestion
  "org.http4s" %% "http4s-otel4s-middleware" % "<version>"
```